include/adaflo/navier_stokes.h

```diff
@@ -168,6 +168,13 @@ class NavierStokes : public FlowBaseAlgorithm<dim>
                            const unsigned int                                face,
                            const double                                      density);
 
+  void
+  set_user_defined_material(std::function<Tensor<2, dim, VectorizedArray<double>>(
+                              const Tensor<2, dim, VectorizedArray<double>> &,
+                              const unsigned int,
+                              const unsigned int,
+                              const bool)> my_user_defined_material);
```

Comment:
Could you document somewhere the meaning of the individual parameters?

Reply:
Of course :facepalm: